backend/src/routes/InvitationRouter.ts

```diff
@@ -0,0 +1,213 @@
+import express, { Request } from "express";
+import { PrismaClient, Status } from "@prisma/client";
+import { StatusCodes } from "http-status-codes";
+import { TripParams } from "./TripRouter";
+
+const router = express.Router({ mergeParams: true });
+const prisma = new PrismaClient();
+
+export interface InvitationParams extends TripParams {
+  invitationId: string,
+  code?: string;
+  status?: string;
+}
+
+// temporary for testing until auth done
+const userID = "6669267e34f4cab1d9ddd751";
+
+// get all invitations
+router.get("/", async (req: Request, res) => {
+  try {
+    const trips = await prisma.tripMember.findMany();
```

Comment:
Rename to `invitations` instead of `trips` to avoid confusion. Same with all below